Vizzuality/widget-editor
# Fix ESLint configuration issues

This PR fixes issues where ESLint would report false positive errors in Typescript files. I believe these issues have been fixed by updating Typescript and fixing the version of ESLint used for linting.

## Testing instructions

Open TS files such as [this one](https://github.com/Vizzuality/widget-editor/blob/4d88daf15785ad07039683ca7e693baa33cf6b8b/src/packages/core/src/services/fields.ts#L1) and make sure they don't contain errors on keywords such as `private`, `public`, `static` or `implements XXX`.

## Pivotal Tracker

Not tracked.
